packages/warn/src/index.ts

```diff
@@ -7,11 +7,11 @@
  * For example: stats warns when any tracks events aren't properly formatted (@see lib/analytics)
  */
 
-let warn;
+let warn: typeof console.warn;
 if ( process.env.NODE_ENV === 'production' || 'function' !== typeof console.warn ) {
-	warn = () => {};
+	warn = (): void => {};
 } else {
-	warn = ( ...args ) => console.warn( ...args );
+	warn = ( ...args: Parameters< typeof console.warn > ): void => console.warn( ...args );
```

Comment:
We can assign the `console.warn` function directly:
```js
warn = console.warn;
```
All `console` methods are self-bound, i.e., don't need the `console.warn()` call convention to get the right `this` pointer.